server/controllers/aidComments.js

```diff
@@ -0,0 +1,152 @@
+/* eslint-disable camelcase */
+/* eslint-disable sort-keys */
+import {
+    INTERNAL_SERVER_ERROR, CREATED, NOT_FOUND, getStatusText
+} from 'http-status-codes';
+
+const db = require('./promise').AidDb;
+
+const averageRating = async aid => {
+    if (aid.comments && aid.comments.length > 0) {
+        const commentCount = await aid.comments.length;
+        let commentTotal = 0;
+        for (let i = 0; i < commentCount; i += 1) {
+            commentTotal += aid.comments[i].rating;
+        }
```

Comment:
```
for (const comment of aid.comments) {
    const { rating } = comment;
    commentTotal += rating;
}
```

Reply:
I am getting this error i don't understand 

> iterators/generators require regenerator-runtime, which is too heavyweight for this guide to allow them. Separately, loops should be avoided in favor of array iterations.

Reply:
yeaaaa, you will need to do some configuration with your next.config.js. A better solution is to just loop over the `comments`
```
comments.forEach(comment => {
    const { rating } = comment;
    commentTotal += rating;
})
```